service/src/main/java/org/broadinstitute/listener/relay/OauthInfo.java

```diff
@@ -1,6 +1,7 @@
 package org.broadinstitute.listener.relay;
 
 import java.time.Instant;
+import java.util.Map;
 import java.util.Optional;
 
-public record OauthInfo(Optional<Instant> expiresAt, String error) {}
+public record OauthInfo(Optional<Instant> expiresAt, String error, Map<String, String> claims) {}
```

Comment:
I am tacking on an extra map of claims here so we can just reach in and get what we need for logging rather than having to break out a separate parameter for each claim.